exoscale/resource_exoscale_sks_cluster.go

```diff
@@ -303,8 +303,8 @@ func resourceSKSClusterCreate(ctx context.Context, d *schema.ResourceData, meta
 		createReq.AutoUpgrade = &autoUpgrade
 	}
 
-	if enableKubeProxy, isSet := d.GetOk(resSKSClusterAttrEnableKubeProxy); isSet {
-		v := enableKubeProxy.(bool)
+	if !d.GetRawConfig().GetAttr("enable_kube_proxy").IsNull() {
```

Comment:
```suggestion
	if !d.GetRawConfig().GetAttr(resSKSClusterAttrEnableKubeProxy).IsNull() {
```
seems like the constant could be reused here